src/sentry/static/sentry/app/components/charts/components/tooltip.jsx

```diff
@@ -12,7 +12,11 @@ function defaultFormatAxisLabel(value, isTimestamp, utc) {
 }
 
 function valueFormatter(value) {
-  return value.toLocaleString();
+  if (typeof value === 'number') {
+    return value.toLocaleString();
+  }
+
+  return value;
```

Comment:
silly nit: would look nice as a ternary too 🤷‍♂️ 

```js
return typeof value === 'number' ? value.toLocaleString() : value;
```

Reply:
I agree, it's a bit silly 🤷‍♂️